projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -458,6 +458,16 @@ public function add_gutenberg_menus() {
 		}
 
 		$this->update_menu( 'gutenberg-edit-site', 'https://wordpress.com/site-editor/' . $this->domain, null, null, null, 59 );
+
+		// Gutenberg 11.9 moves the Site Editor to an Appearance submenu as Editor.
+		$submenus_to_update = array(
```

Comment:
Curious: Since Jetpack is used in pure org sites too, AFAIU, there are sites that are JP-connected sites that are outside of the actual WPCOM scope. Would hardcoding `wordpress.com` here be a problem? Or is that because the feature is still in a testing phase and only available in WPCOM?

Reply:
This is a good point, as I think this will break on wpcalypso and horizon environments, by linking to production? Not ideal as we tend to point people to horizon to specifically sign up for the FSE beta.

There's some prior art here that might be helpful: https://github.com/Automattic/jetpack/blob/e31d94b4037befdd9a680397e9c51766bb2c20f6/projects/plugins/jetpack/modules/calypsoify/class-jetpack-calypsoify.php#L102

That said, this is currently hard-coded today, so I don't think this should block the PR. We could handle in a follow-up, if needed.

Reply:
@fullofcaffeine Great question! Non-Atomic Jetpack sites do not get Nav Unification. Their links are left as-is.

Reply:
> as I think this will break on wpcalypso and horizon environments, by linking to production? Not ideal as we tend to point people to horizon to specifically sign up for the FSE beta.

Im not sure if this is the case.  Just in testing but  If I am running in horizon and use the link I still stay in horizon.  If thats what you were getting at?

Reply:
You're right, it's working correctly in horizon for me and links to `https://horizon.wordpress.com/site-editor/{site}` from the new sidebar menu location. So it looks like we're in the clear and this is fine; sorry for the false alarm!